From f7f12d4c17a3136494af0d660fcc284e2d848f78 Mon Sep 17 00:00:00 2001
From: user189 <shiva.kumara.rudrappa@intel.com>
Date: Thu, 18 Mar 2021 16:27:24 +0530
Subject: [PATCH] Captured image show black in image view

with android 11 and target sdk version above 29 files
not allowed to share with external source. to do this
need additional access.

Solution : added additional access of MANAGE_EXTERNAL_STORAGE
to access all media files.

Change-Id: Icf0dfe46fe5d328f5da6e9debd207d3e86a01333
Tracked-On: OAM-96545
Signed-off-by: shiva kumara R <shiva.kumara.rudrappa@intel.com>
---
 AndroidManifest.xml | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/AndroidManifest.xml b/AndroidManifest.xml
index 4d7e17144..d2c35f2b9 100644
--- a/AndroidManifest.xml
+++ b/AndroidManifest.xml
@@ -28,6 +28,7 @@
     <uses-permission android:name="android.permission.WAKE_LOCK" />
     <uses-permission android:name="android.permission.WRITE_SETTINGS" />
     <uses-permission android:name="android.permission.WRITE_SYNC_SETTINGS" />
+    <uses-permission android:name="android.permission.MANAGE_EXTERNAL_STORAGE"/>
     <uses-permission android:name="android.permission.POST_NOTIFICATIONS" />
 
     <supports-screens
@@ -46,6 +47,7 @@
         android:restoreAnyVersion="true"
         android:supportsRtl="true"
         android:theme="@style/Theme.Camera"
+        android:requestLegacyExternalStorage="true"
         android:usesCleartextTraffic="false" >
         <activity
             android:name="com.android.camera.CameraActivity"
-- 
2.17.1

